exercise/static/exercise/assessment.js

```diff
@@ -0,0 +1,98 @@
+$(function () {
+  // Create links for switching between assistant feedback and grader
+  // feedback in the assessment bar
+  var feedback1 = $('#id_assistant_feedback').closest('.form-group');
+  var feedback2 = $('#id_feedback').closest('.form-group');
+  feedback1.addClass('feedback-toggle');
+  feedback2.addClass('feedback-toggle hide');
+  var label1 = feedback1.find('label');
+  var label2 = feedback2.find('label');
+  var feedbackToggleLink = $('<a href="#"></a>')
```

Comment:
> For actions that modify the current page content like switching tabs or hiding a section, use <button>

Bootstrap uses `<a>` elements for tabs (and "pills"). If we change them into buttons because we would then lose Bootstrap's default styles for tabs/pills.

The feedback toggle link that you pointed out here _can_ be changed to a button since it's not a tab element, but there's a pill element in _assessment_panel.html that is used for switching between grader feedback and grader errors. What should I do about that? I guess I could turn it into buttons and use the new visibility switching functionality instead of Bootstrap's pills.

Reply:
The "close sidebar" link is also a pill element. I could turn it into a button, but then it would stick out like sore thumb:
![image](https://user-images.githubusercontent.com/28843787/135863350-da798ca5-c48f-4063-b686-9259798b2ce2.png)
Or maybe I could make the button smaller and put it next to the language switch button?

Reply:
> Or maybe I could make the button smaller and put it next to the language switch button?

![image](https://user-images.githubusercontent.com/28843787/135990875-c76e3b2f-be44-44ca-8bdd-7780af9b3dfe.png)
Looks pretty good in my opinion. I also managed to make the sidebar expander element a button without altering its appearance.

Reply:
Sorry, I didn't realize that Bootstrap 3 has this limitation. We don't need to make this too complicated. If it is not trivial (and quick to finish) to change `<a>` to `<button>`, then don't spend time on it. If you already did something, that's fine too. If you need to use `<a>` for an action (that is not a link to another page), then it should have `role="button"`.

The screenshot of the new "close sidebar" button looks good to me.

Reply:
Fixed, the feedback toggles are now buttons.